lib/lure/versionManager/mvn/mvn.go

```diff
@@ -86,7 +86,7 @@ func MvnOutdated(path string) (error, []versionManager.ModuleVersion) {
 func getModulePropertyMap(path string) map[string]string {
 	var moduleProperties map[string]string = make(map[string]string)
 
-	cmd := exec.Command("mvn", "test-compile", "compile", "-B", "-q", "--also-make", "exec:exec", "-Dexec.executable=pwd")
+	cmd := exec.Command("mvn", "test-compile", "compile", "-B", "-U", "-q", "--also-make", "exec:exec", "-Dexec.executable=pwd")
```

Comment:
Why not use full options anywhere? IMO it's clearer in the end.

Reply:
done